packages/zowe-explorer-api/src/profiles/ZoweExplorerApi.ts

```diff
@@ -331,6 +331,17 @@ export namespace ZoweExplorerApi {
          * @returns {Promise<zowe.IZosFilesResponse>}
          */
         dataSetsMatchingPattern?(filter: string[], options?: zowe.IDsmListOptions): Promise<zowe.IZosFilesResponse>;
+
+        /**
+         * Copies a dataSet.
+         *
+         * @param {string} fromDataSetName
+         * @param {string} toDataSetName
+         * @param {string?} enq possible values : {SHR, SHRW, EXCLU}
+         * @param {boolean?} replace
+         * @returns {Promise<zowe.IZosFilesResponse>}
+         */
+        copyDataSet(fromDataSetName: string, toDataSetName: string, enq?: string, replace?: boolean): Promise<zowe.IZosFilesResponse>;
```

Comment:
Don't we need to make this optional to avoid breaking extenders?

Reply:
Good catch ! adressed in : https://github.com/zowe/vscode-extension-for-zowe/pull/2051/commits/b62c2e4eed34167aa8faa83bb2baee2228ebc82d